io_scene_godot/converters/physics.py

```diff
@@ -219,9 +219,12 @@ def export_physics_controller(escn_file, export_settings, node,
 
     phys_obj = NodeTemplate(phys_name, phys_controller, parent_gd_node)
 
-    #  OPTIONS FOR ALL PHYSICS TYPES
-    phys_obj['friction'] = rbd.friction
-    phys_obj['bounce'] = rbd.restitution
+    if phys_controller != 'KinematicBody':
+        physics_mat = InternalResource("PhysicsMaterial")
```

Comment:
IMO the resource name is still helpful here 
![image](https://user-images.githubusercontent.com/19422835/99278735-66b95b00-286a-11eb-9e8a-6fc55a7dd5d5.png)
If there is no good choice, we can just use `phys_name + "PhysicsMaterial"` 😊

Reply:
Will do. I think I just copied from a PhysicsMaterial I created in Godot, which is unnamed by default, but might as well give it a name if we can 👍